README.md

```diff
@@ -1,50 +1,96 @@
-# Universal Permissionless Rewards Distributor
+# Universal Rewards Distributor
+The Universal Rewards Distributor (URD), is a smart contract that allows for the distribution of multiple ERC20 tokens from a single off-chain computed Merkle tree.
 
-A universal permissionless rewards distributor written in Solidity. It allows the distribution of any reward token (different reward tokens are possible simultaneously) based on a Merkle tree distribution, and using ERC20 allowance of a treasury.
+Each URD contract has an owner and a group of updaters. These updaters can submit a new root to a pending value, which is subject to a timelock (that can be set to 0). Once the timelock period ends, anyone can accept this pending value. However, during the timelock period, the owner has veto power and can remove the pending value.
```

Comment:
I think the timelock should be introduced first, to avoid making nested sentences like "which is subject to a timelock (that can be set to 0)"
Nested/parentheses sentences are harder to read than simple, linear sentences